test/e2e-common.sh

```diff
@@ -370,7 +370,14 @@ function run_postinstall_jobs() {
 function uninstall_channel_crds() {
   echo "Uninstalling Kafka Channel CRD"
   kubectl delete secret -n "${SYSTEM_NAMESPACE}" kafka-cluster
-  sleep 10 # Give Controller Time To React To Kafka Secret Deletion ; )
+
+  # The distributed channel controller no longer actively monitors the kafka-cluster secret,
+  # so the receiver will only be torn down if there is a change to the kafkachannels or the
+  # config-kafka configmap.  However, if the secret is missing, the global resync does not happen
+  # (the controller presumes this missing secret is an error and reacts accordingly), so this
+  # manual deletion of the receiver deployment is the easiest solution here.
+  [[ $1 == "distributed" ]] && kubectl delete deployment -n "${SYSTEM_NAMESPACE}" kafka-cluster-32603413-receiver
+
```

Comment:
do we need to delete the receiver Service as well ?

Reply:
That would be a good idea; I forgot about that.  Distributed is the last test before the cluster gets torn down anyway but it's good to be clean.